### ISSUE TITLE
Indexer crashes when listing accounts

### ISSUE THREAD

**Issue by n1tranquilla** [Granola-Team/mina-indexer#915]
```
curl 'http://api.minasearch.com/graphql' \
  -H 'sec-ch-ua: "Chromium";v="124", "Brave";v="124", "Not-A.Brand";v="99"' \
  -H 'sec-ch-ua-platform: "macOS"' \
  -H 'Referer: http://127.0.0.1:5173/' \
  -H 'DNT: 1' \
  -H 'sec-ch-ua-mobile: ?0' \
  -H 'User-Agent: Mozilla/5.0 (Macintosh; Intel Mac OS X 10_15_7) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/124.0.0.0 Safari/537.36' \
  -H 'content-type: application/json' \
  --data-raw $'{"variables":{"limit":100,"sortBy":"BALANCE_DESC","query":null},"query":"query AccountsQuery(\\n  $limit: Int = 100\\n  $sortBy: AccountSortByInput\u0021\\n  $query: AccountQueryInput\\n) {\\n  accounts(limit: $limit, query: $query, sortBy: $sortBy) {\\n    publicKey\\n    username\\n    balance\\n    delegate\\n    nonce\\n    timeLocked\\n  }\\n}\\n","operationName":"AccountsQuery"}'
```

**Comment by Isaac-DeFrain** [comment]
I believe this will be fixed by #916

**Comment by n1tranquilla** [comment]
Was able to spam using this cURL with no issues
